docs/body.md

```diff
@@ -459,6 +459,16 @@ This ZenPack still supports the previous configuration, where messages were sent
 
 For new openstack installs, we recommend using the RabbitMQ-Ceilometer endpoint, which is what will be reported by `openstack_amqp_config`.
 
+When the RabbitMQ-Ceilometer service is added to Control Center, two IP Assignments will be added as well.  Upon creation, they will have no IP address, only a port number.  You must add the IP address by following these steps:
```

Comment:
Ah, looks like one is the regular RabbitMQ node and the other is the management console.  Very good.  I'll add that in.